crates/typst-library/src/text/raw.rs

```diff
@@ -126,6 +132,49 @@ pub struct RawElem {
     /// ````
     #[default(HorizontalAlign(GenAlign::Start))]
     pub align: HorizontalAlign,
+
+    /// A set of additional syntax definitions to load.
+    /// The syntax definitions should be in the `sublime-syntax` file format.
+    /// Syntaxes are a dictionary of the language name and the syntax file.
+    ///
+    /// ````example
+    /// #set raw(syntaxes: "SExpressions.sublime-syntax")
+    ///
+    /// ```sexp
+    /// (defun factorial (x)
+    ///    (if (zerop x)
+    ///        ; with a comment
+    ///        1
+    ///        (* x (factorial (- x 1)))))
+    /// ```
+    /// ````
+    #[parse(
+        let (syntaxes, data) = if let Some(Spanned { v: paths, span }) = args.named::<Spanned<SyntaxPaths>>("syntaxes")? {
```

Comment:
auto format doesn't work in the macro. could you format it somehow? e.g. by pasting it somewhere else where the indent is right and then pasting it back.